pkg/operator/ceph/cluster/mon/config.go

```diff
@@ -39,6 +38,40 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+/* TODO: clean up keyring secrets when cluster is torn down? */
```

Comment:
Nope. That's great. 

Looking ahead, I think for other daemons it will be necessary to delete keyring secrets when daemons are destroyed to prevent resource leakage. Like when an mds scales down, for example. But if that owner reference can be set to the deployment somehow, that would be good.

Reply:
yes, owner reference is nice that way to make sure the resource gets cleaned up when the parent gets deleted